libbpf-tools/runqslower.bpf.c

```diff
@@ -99,13 +116,19 @@ int BPF_PROG(sched_switch, bool preempt, struct task_struct *prev, struct task_s
 SEC("raw_tp/sched_wakeup")
 int BPF_PROG(handle_sched_wakeup, struct task_struct *p)
 {
-	return trace_enqueue(BPF_CORE_READ(p, tgid), BPF_CORE_READ(p, pid));
+	char comm[TASK_COMM_LEN] = {0};
+
+	BPF_CORE_READ_STR_INTO(&comm, p, comm);
```

Comment:
We have to do string copy even if `targ_comm` is not set.
Consider adding a feature flag, if not set, pass `NULL` instead.